lib/tasks/lumen.rake

```diff
@@ -255,6 +255,29 @@ namespace :lumen do
     end
   end
 
+  desc 'Populate domain_counts table'
+  task populate_domain_count: :environment do
+
+    def give_domains_from_urls(infringing_urls_list)
```

Comment:
...and *this* is the point where I've seen this function twice, and now it makes sense to think about where it wants to live -- because we definitely want it to be the same logic in all places we are using it.

I think this wants to be a class method on `DomainCount`. There's a couple of ways you could go about that:
* leave it where it is in the other PR; once that is merged, rebase this branch onto the new dev and refactor
* define it on DomainCount in the other branch; cherry-pick that commit into here; update `NoticeSubmissionFinalizer` in the other branch accordingly

Maybe some other options? At any rate, now we know where it wants to live, so let's make it live there and ensure both branches end up in sync.